sdk/identity/azure-identity/azure/identity/_credentials/certificate.py

```diff
@@ -54,9 +56,30 @@ def __init__(self, tenant_id, client_id, certificate_path, **kwargs):
 
         # TODO: msal doesn't formally support passwords (but soon will); the below depends on an implementation detail
         private_key = serialization.load_pem_private_key(pem_bytes, password=password, backend=default_backend())
+        client_credential = {"private_key": private_key, "thumbprint": hexlify(fingerprint).decode("utf-8")}
+        if kwargs.pop("send_certificate", False):
```

Comment:
Should we do this only in
if kwargs.pop("send_certificate", False):
else:
    private_key = serialization.load_pem_private_key(pem_bytes, password=password, backend=default_backend())
    client_credential = {"private_key": private_key, "thumbprint": hexlify(fingerprint).decode("utf-8")}

Reply:
We need the private key and fingerprint in both cases. `send_certificate=True` means we need to include the cert content as well.